### PR TITLE
Map Block: A8C Mapbox Account

### PR DESCRIPTION
**Pull request by jeffersonrabb**
#### Changes proposed in this Pull Request

The original Map block used styles associated with my personal Mapbox.com account. This PR switches to A8C's account. 

#### Testing instructions

Test locally or with JN, and verify that map styles continue to work.

JN link: https://jurassic.ninja/create/?gutenberg&gutenpack&shortlived&jetpack-beta&calypsobranch=update/map-a8c-mapbox-account&branch=master

### CODE REVIEW

File: client/gutenberg/extensions/map/mapbox-map-formatter/index.js [hunk 1]
```diff
@@ -4,20 +4,20 @@
 export function mapboxMapFormatter( mapStyle, mapDetails ) {
```
Comment (simison) [inline client/gutenberg/extensions/map/mapbox-map-formatter/index.js]: We can remove this "/* TODO: Replace with styles created in A8C-owned account */" line now

**Comment by sirreal** [comment]
What am I looking for to confirm this is working well?

I am able to insert the block and change between the 4 styles in the block. Everything appears to render well.

File: client/gutenberg/extensions/map/mapbox-map-formatter/index.js (hunk 1 above)
Comment (simison) [reply client/gutenberg/extensions/map/mapbox-map-formatter/index.js]: Ah sorry, this was meant to be comment for line 3

Comment (jeffersonrabb) [reply client/gutenberg/extensions/map/mapbox-map-formatter/index.js]: Removed in https://github.com/Automattic/wp-calypso/pull/28729/commits/ab4af733c103ca97b88fc6875e413d65df8ad574.

**Pull request merged**